pycross/private/tools/bzl_lock_generator.py

```diff
@@ -770,6 +783,20 @@ def main(args: Any) -> None:
         else:
             target.build_deps.append(resolved_dep)
 
+    for ignore_dependency in args.ignore_dependency or []:
+        pkg, dep = ignore_dependency
+        resolved_pkg = resolve_single_version(
+            pkg,
+            package_keys_by_canonical_name,
+            package_targets_by_package_key.keys(),
+            "package_build_dependencies",
```

Comment:
This should be updated to `package_ignore_dependencies` so the error message is correct.

Reply:
done